src/home/WalletHome.tsx

```diff
@@ -115,6 +117,9 @@ function WalletHome() {
   }
 
   const shouldShowCashInBottomSheet = () => {
+    if (!isFocused) {
+      return false
+    }
```

Comment:
one side effect of this change is that the bottom sheet will now appear every time the home screen gets back focus and the other conditions satisfy, whereas previously it appears only once on app load and then later only once when the balance goes from non zero to zero. I believe this is tracked by local state and this state is kept in memory as long as the conditions to show / not show never changes. With this change though, I believe we lose this state every time focus changes.

Reply:
🤔 We could move the cash-in bottom sheet into redux state and dispatch it similar to to the in-app review.

Reply:
@satish-ravi I ended up wrapping the component in a useMemo which seems to do the trick in https://github.com/valora-inc/wallet/pull/3927/commits/500d92fff21640fd6991334543e706821bcecd87. I'll test a bit more and confirm things are WAI.